CHANGELOG.md

```diff
@@ -22,6 +22,9 @@ _None_
   [nicorichard](https://github.com/nicorichard)
   [#913](https://github.com/SwiftGen/SwiftGen/issues/913)
   [#926](https://github.com/SwiftGen/SwiftGen/pull/926)
+* Strings: added support for `.strings` files comments. The built-in templates will now use them for comments instead of the translation of a key.
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/SwiftGen/SwiftGen/blob/acf2297afff17ac0a566ce10f64b50e40bdb8b4b/CHANGELOG.md#L25"></span>Line describing your entry should end with a period and 2 spaces.</td>
    </tr>
  </tbody>
</table>